### PR TITLE
Add latest RALI changes

### PR DESCRIPTION
**Pull request by LokeshBonta**

### CODE REVIEW

**Comment by kiritigowda** [comment]
@LokeshBonta can you sync this with TOT master. This PR is removing the changes which we have made.

**Comment by kiritigowda** [comment]
@LokeshBonta can you also check on [LGTM Warnings](https://lgtm.com/projects/g/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/rev/pr-d4b7c76bbab5c7075b0c6e574372395ff7766702)

**Comment by kiritigowda** [comment]
@LokeshBonta this PR fails build test. check [Travis](https://travis-ci.org/github/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/builds/688435477)

```
[ 98%] Building CXX object rali/CMakeFiles/rali.dir/source/reader_factory.cpp.o
1430In file included from /MIVisionX/rali/./include/tf_record_reader.h:11:0,
1431                 from /MIVisionX/rali/source/reader_factory.cpp:4:
1432/MIVisionX/rali/./tf/example.pb.h:10:10: fatal error: google/protobuf/port_def.inc: No such file or directory
1433 #include <google/protobuf/port_def.inc>
1434          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1435compilation terminated.
1436rali/CMakeFiles/rali.dir/build.make:1694: recipe for target 'rali/CMakeFiles/rali.dir/source/reader_factory.cpp.o' failed
1437make[2]: *** [rali/CMakeFiles/rali.dir/source/reader_factory.cpp.o] Error 1
1438CMakeFiles/Makefile2:844: recipe for target 'rali/CMakeFiles/rali.dir/all' failed
1439make[1]: *** [rali/CMakeFiles/rali.dir/all] Error 2
1440Makefile:151: recipe for target 'all' failed
1441make: *** [all] Error 2
```

File: rali/README.md [hunk 1]
```diff
@@ -63,14 +71,32 @@ cmake -DCMAKE_INSTALL_PREFIX=/usr \
...
 sudo make install      
 ```
 
-##Jsoncpp installation
+## Jsoncpp installation
 
 sudo apt-get install libjsoncpp-dev
 
+## Google Protobuf installation
+https://github.com/protocolbuffers/protobuf/blob/master/src/README.md
```
Comment (kiritigowda) [inline rali/README.md:83]: Currently MIVisionX installs [protobuf v3.5.2](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX#tested-configurations)

Comment (LokeshBonta) [reply rali/README.md:83]: Okay. We will remove them

**Comment by LokeshBonta** [comment]
@kiritigowda 
 The pb files we use from tensorflow requires protobuf version 3.11.4
That is why we have included that in the Readme.md

File: rali/README.md (hunk 1 above)
Comment (LokeshBonta) [reply rali/README.md:83]: The pb files we use from tensorflow requires protobuf version 3.11.4

**Comment by LokeshBonta** [comment]
LGTM warning have been resolved now with the latest commit

**Comment by kiritigowda** [comment]
> @kiritigowda
> The pb files we use from tensorflow requires protobuf version 3.11.4
> That is why we have included that in the Readme.md
> 
> Can we change the default installation to 3.11.4?

@LakshmiKumar23 can we check if the protobuf version is set to 3.11.4 MIVisionX can still build? @LokeshBonta

**Comment by LakshmiKumar23** [comment]
@kiritigowda MIVisionX builds with libprotoc 3.11.4.

**Comment by kiritigowda** [comment]
@LokeshBonta you first need to sync your master to MIVisionX/master. I just saw that your master is 19 commits behind the TOT master.

File: rali/include/bounding_box_graph.h [hunk 2]
```diff
@@ -1,25 +1,3 @@
-/*
```
Comment (kiritigowda) [inline rali/include/bounding_box_graph.h:1]: Overwriting changes in master

**Comment by LokeshBonta** [comment]
Hi
@kiritigowda @rrawther 
I will create another pull request with some other modifications which we talked about in the call
Please let me know if it is okay to close the pull request.

**Comment by kiritigowda** [comment]
@LokeshBonta yes, we can close this PR. I will close this PR and you can open another with the changes.

**Pull request closed without merging**